impl/src/main/java/org/ehcache/internal/persistence/DefaultLocalPersistenceService.java

```diff
@@ -77,12 +79,13 @@ public synchronized void stop() {
         // passes on windows
         rw.close();
         if (!lockFile.delete()) {
-          // todo log something?;
+          LOGGER.info("Lock file was not deleted {}.", lockFile.getPath());
```

Comment:
Since this method can throw exceptions on error, why shouldn't this also throw an exception too instead of logging the error?